bin/solaar

```diff
@@ -26,7 +26,23 @@ def init_paths():
 	import sys
 	import os.path as _path
 
-	prefix = _path.normpath(_path.join(_path.realpath(sys.path[0]), '..'))
+	# Python 2 need converstion from utf-8 filenames
+	# Python 3 might have problems converting back to UTF-8 in case of Unicode surrogates
+	try:
+	    if sys.version_info < (3,):
+	    	decoded_path = sys.path[0].decode(sys.getfilesystemencoding())
+	    else:
+	    	decoded_path = sys.path[0]
+	    	teste = sys.path[0].encode(sys.getfilesystemencoding())
+
+	except UnicodeDecodeError:
+	    sys.stderr.write('ERROR: Solaar cannot decode filesystem path, this may happen because non UTF-8 characters in the pathname.\n')
+	    sys.exit(1)
+	except UnicodeEncodeError:
+	    sys.stderr.write('ERROR: Solaar cannot encode filesystem path, this may happen because non UTF-8 characters in the pathname.\n')
+	    sys.exit(1)
```

Comment:
Both are subclasses of UnicodeError (see `pydoc2 UnicodeEncodeError` and `pydoc3 UnicodeDecodeError`). (Alternatively, `except UnicodeEncodeError | UnicodeDecodeError` will also catch both).